src/listener/handlers/document-update-handler.ts

```diff
@@ -152,32 +150,35 @@ class DocumentUpdateHandler implements IHandler {
     checkEligibleForFileValidation(contentValidationReport: ValidationReport, documentUrl: string, documentType: string): Promise<ValidationReport> {
         if (documentType.toUpperCase() === 'FILE') {
             // refresh doc before checking cloudUrl
-            return this.ingestClient.retrieveMetadataDocument(documentUrl).then((doc: any) => {
-                if (doc['cloudUrl']) {
-                    // if cloud url present, proceeds with file validation
-                    if (contentValidationReport.validationState.toUpperCase() == "VALID") {
-                        return this.attemptFileValidation(contentValidationReport, doc, documentType)
+            return this.ingestClient.retrieveMetadataDocument(documentUrl)
```

Comment:
Async/await

Reply:
what do you think?

Reply:
yeah, I think there's no need for this now... the if clauses are simpler now